src/microcosm.js

```diff
@@ -177,7 +180,14 @@ export default class Microcosm extends Emitter {
       config = { register: config }
     }
 
-    this.stores = this.stores.concat([[ key, config ]])
+    this.domains = this.domains.concat([[ key, config ]])
+
+    for (const domain of this.domains) {
+      const [key, config] = domain
+      if (key) {
```

Comment:
A warning feels appropriate. Maybe there is a legitimate use case for overwriting a domain?